src/configuration/js-team/index.js

```diff
@@ -0,0 +1,35 @@
+import Linux64 from './linux64';
```

Comment:
Try to see if you can follow this style (both named exactly the same):
`import foo from './foo';`